content/docs/aws/deploy/install-kubeflow.md

```diff
@@ -51,9 +50,6 @@ Your Kubeflow `app` directory contains the following files and directories:
     * This directory is created when you run `kfctl.sh generate platform`.
     * You can modify the `cluster_config.yaml` and `cluster_features.sh` files to customize your AWS infrastructure.
 * **${KFAPP}/k8s_specs** - A directory that contains YAML specifications for daemons deployed on your Kubernetes Engine cluster.
-* **${KFAPP}/ks_app** - A directory that contains the [ksonnet](https://ksonnet.io/) application for Kubeflow.
```

Comment:
Can you add one line here about the kustomize file directory? See https://github.com/kubeflow/website/pull/887/files